tests/tests/test_testcase.py

```diff
@@ -1,20 +1,151 @@
 from tests import TestCase
+from masoniteorm.models import Model
 from src.masonite.routes import Route
+from src.masonite.auth import Authenticates
+from src.masonite.auth import Auth
+from src.masonite.auth.guards import WebGuard
+
+
+class User(Model, Authenticates):
+    pass
 
 
 class TestTesting(TestCase):
     def setUp(self):
         super().setUp()
-        self.addRoutes(Route.get("/test", "WelcomeController@show").name("test"))
+        self.addRoutes(
+            Route.get("/test", "WelcomeController@show").name("test"),
```

Comment:
Here I am adding the test routes, but when I run tests the response for each route is always `b'welcome'`